src/utils/metadata.ts

```diff
@@ -0,0 +1,30 @@
+export const KEY = 'es-model';
```

Comment:
if not used anywhere else, I'd hide it (don't export it). You wrote the helpers for that.